jans-cedarling/test_files/policy-store_ok.json

```diff
@@ -0,0 +1,56 @@
+cedar_version: v4.0.0
```

Comment:
The fn is policy-store_ok.json ... but it looks like YAML?

Reply:
i meant to convert that file into JSON from the YAML example but ended up not using it... then forgot to delete it 😅

file now deleted in [2c1d0df](https://github.com/JanssenProject/jans/pull/10216/commits/2c1d0dff3576c957e2d59ca029a67ac1729c0c93)